src/universal/components/LogoBlock/LogoBlock.js

```diff
@@ -28,7 +28,7 @@ const LogoBlock = (props) => {
   const {variant} = props;
   const logoSrc = variant === 'primary' ? logoMarkPrimary : logoMarkWhite;
   return (
-    <Block>
```

Comment:
`variant` can be `primary` or `white` but `white` isn't handled anywhere

Reply:
Not sure I know what you mean, but the white variant is used on the dashboard sidebar and the primary variant is used on the meeting sidebar. I set image color, and piggy backed on that prop to add a border for the meeting sidebar. It’s working as intended so far.